commit 547ffb4b0da6260765b2bcdb96d8c014d579c7a3 (refs/changes/55/4155/1)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-06-05T14:18:45-04:00 (1 year, 4 months ago)
    
    TRITON-461 triton services still on nodejs 0.10.x should at least use the latest x
    TRITON-464 sdc-napi-ufds-watcher NODE_PREBUILT_IMAGE does not match runtime environment

diff --git a/Makefile b/Makefile
index 4389e08..7ebe6d2 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2015, Joyent, Inc.
+# Copyright (c) 2018, Joyent, Inc.
 #
 
 #
@@ -37,11 +37,10 @@ JSSTYLE_FLAGS    = -o indent=4,doxygen,unparenthesized-return=0
 JSON_FILES  := package.json
 SMF_MANIFESTS_IN = smf/manifests/napi-ufds-watcher.xml.in
 
-NODE_PREBUILT_VERSION=v0.10.29
+NODE_PREBUILT_VERSION=v0.10.48
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_TAG=zone
-	# Allow building on a SmartOS image other than sdc-smartos@1.6.3.
-	NODE_PREBUILT_IMAGE=fd2cc906-8938-11e3-beab-4359c665ac99
+	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
 endif
 
 include ./tools/mk/Makefile.defs
